pkg/apis/crd/v1alpha2/types.go

```diff
@@ -210,6 +210,11 @@ type EgressSpec struct {
 	AppliedTo AppliedTo `json:"appliedTo"`
 	// EgressIP specifies the SNAT IP address for the selected workloads.
 	EgressIP string `json:"egressIP"`
+	// ExternalIPPool specifies the IP Pool that the EgressIP should be allocated from.
+	// If it's set, the allocated EgressIP will be assigned to a Node specified by the Pool and it will failover to
+	// another Node if the assigned Node becomes unavailable.
+	// If it's set with EgressIP, the EgressIP must be in the IP Pool.
+	ExternalIPPool string `json:"externalIPPool"`
```

Comment:
nit: maybe replace "is set" with "is not the empty string" for the sake of clarity?

Reply:
rephrase the comments of EgressIP and ExternalIPPool, PTAL.